webapp/src/utils/validationUtils.ts

```diff
@@ -4,16 +4,19 @@ import { t } from "i18next";
 // Types
 ////////////////////////////////////////////////////////////////
 
-interface ValidationOptions {
+interface NumberValidationOptions {
+  min?: number;
+  max?: number;
+}
+
+interface ValidationOptions extends NumberValidationOptions {
```

Comment:
Consolidating shared properties like min and max into a single interface eliminates redundancy but keep in mind the context, here their serve different purposes. I think we should keep them separate to simplify future additions and flexibility

Reply:
I forgot to commit a fix for the length message sorry.